src/contents/articles/hello-world/index.md

```diff
@@ -1,12 +1,10 @@
 ---
-title: Hello World
+title: Introducing Namespace
 author: Marcus Sanatan
 date: 2015-05-24
 template: article.jade
 ---
 
-### This is **namespace**
-
 Hi everyone, we're officially launching namespace - a club for software developers in Trinidad and Tobago. We're a movement to remove the silos of people scattered throughout the nation that prevents us from build a community. 
 
 ## Why do we need a community?
```

Comment:
I disagree, I think they help logically separate the content of the post. Especially for something like the 5 pillars, I believe that sectioning the post gives it the attention it deserves. Any other thoughts?